examples/svm/plot_oneclass_vs_svdd.py

```diff
@@ -0,0 +1,103 @@
+"""
+=========================
+One-Class SVM versus SVDD
+=========================
+
+An example comparing the One-Class SVM and SVDD models for novelty
+detection.
+
+:ref:`Support Vector Data Description (SVDD) <svm_outlier_detection>`
+and :ref:`One-Class SVM <svm_outlier_detection>` are unsupervised
+algorithms that learn a decision function for novelty detection, i.e
+the problem of classifying new data as similar or different to the
+training set.
+
+It can be shown that the One-Class SVM and SVDD models yield identical
+results in the case of a stationary kernel, like RBF, but produce different
+decision functions for non-stationary kernels, e.g. polynomial. This
+example demonstrates this.
+
+Note that it is incorrect to say that the SVDD is equivalent to the
+One-Class SVM: these are different models, which just happen to coincide
+for a particular family of kernels.
+"""
```

Comment:
Much like in case of OneClassSVM `\nu` lower bounds the fraction of support vectors and upper bounds the fraction of outliers in the train.

I varied `\nu = 0.1 and 0.8` in `examples/svm/plot_oneclass_vs_svdd.py` and the decision boundary shrank when `\nu` was higher. The effect was more prominent when I reduced the size of the train sample.

Reply:
Alright, let's keep the example as it is.